bookshelf-standard/4-auth/pom.xml

```diff
@@ -116,8 +116,8 @@ Copyright 2016 Google Inc.
 
     <dependency>                        <!-- Google Cloud Client Library for Java -->
       <groupId>com.google.cloud</groupId>
-      <artifactId>google-cloud</artifactId>
-      <version>0.5.1</version>
+      <artifactId>google-cloud-storage</artifactId>
+      <version>0.10.0-beta</version>
```

Comment:
Huh... I don't see it in maven central yet.. propagation delay?

http://search.maven.org/#search%7Cgav%7C1%7Cg%3A%22com.google.cloud%22%20AND%20a%3A%22google-cloud%22